Library/Formula/graphviz.rb

```diff
@@ -43,7 +43,7 @@ class Graphviz < Formula
 
   if build.with? "bindings"
     depends_on "swig" => :build
-    depends_on :python
+    depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Ping on this. We're trying to clear out the remaining legacy-homebrew issues. Are you still interested in pursuing this?